Makefile.PL

```diff
@@ -0,0 +1,31 @@
+use 5.006;
+use strict;
+use warnings;
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+    NAME             => 'Data::Decimate',
+    AUTHOR           => q{Binary.com <support@binary.com>},
+    VERSION_FROM     => 'lib/Data/Decimate.pm',
+    ABSTRACT_FROM    => 'lib/Data/Decimate.pm',
+    LICENSE          => 'artistic_2',
+    PL_FILES         => {},
+    MIN_PERL_VERSION => 5.006,
```

Comment:
The oldest version we test with is 5.14. However, we claim that it works with 5.6. That is bold.